command.py

```diff
@@ -9,15 +9,11 @@
 
 from typing import List
 
-import configuration
-import database
 import emoji
-import fetcher
-import oracle
 import price
-import rotation
 
-from card import Card
+from magic import configuration, database, oracle, fetcher, rotation
+from magic.card import Card
```

Comment:
I have sympathy for the position that `from X import Y` is a bad practice. If this were magic.card we could have local variables called card without the linter complaining.

Then again magic.card.blah.whatever gets old very quickly. So I could see us going either way. This project is small enough that we don't have huge issues with this kind of import and we have a bunch of top level modules even after this refactor so I'm not sure it matters.